homeassistant/components/ondilo_ico/api.py

```diff
@@ -35,7 +36,7 @@ def refresh_tokens(self) -> dict:
 
         return self.session.token
 
-    def get_all_pools_data(self) -> dict:
+    def get_all_pools_data(self) -> list[dict[str, Any]]:
```

Comment:
The return type is actually a list as far as I can tell.
https://interop.ondilo.com/docs/api/customer/v1/#pools-data-list